EnglishFilter/sections/foreign.txt

```diff
@@ -5636,6 +5636,8 @@ v.kakao.com#$?#ul[class^="list_column"] > li[adfit-unit-id] { remove: true; }
 web.humoruniv.com###wrap_cnts a[href*="/ad_read_"][onmouseover^="ad_choo_"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/84644
 ||ads.priel.co.kr^$third-party
+! https://github.com/AdguardTeam/AdguardFilters/issues/85309
+ohli24.net##.insty > div.nlin > div.exo2
```

Comment:
It's better to block these gifs. You can inspect them via Network, I found one with a size of 1.3 Mb.
`||ohli24.net/img/*/*.gif`, also remove a leftover `ohli24.net###exoall`.